usr/libexec/gamescope-sdl-workaround

```diff
@@ -3,20 +3,22 @@
 
 # Regexes for GPUs requiring the SDL backend workaround
 # Can be used in regexes as $POLARIS_RE|$VEGA_RE etc for readability
-# Needed output for a regex can gathered from "switcherooctl list"
-POLARIS_RE="Baffin|Ellesmere|Lexa PRO|Polaris \d{2} XL|RX 460/560D / Pro 450/455/460/555/555X/560/560X|RX 470/480/570/570X/580/580X/590|540/540X/550/550X|RX 540X/550/550X"
+# Needed output for a regex can gathered from "lspci -nnk | grep -P "(VGA compatible|3D) controller" -A1"
+# The regex format is Perl REGEX
+POLARIS_RE="Ellesmere|Lexa (\[|XT|XL|PRO)|Baffin|Polaris \d{2}|Polaris\d{2}|RX 470/480/570/570X/580/580X/590|540/540X/550/550X|RX 540X/550/550X|RX\s?[4-5]\d{2}(\s|\])"
```

Comment:
The last generic match won't match the `RX 4XX/4YY` or `RX 560X` cases etc.

I think this would be cleaner:
Ellesmere|Lexa|Baffin|Polaris\s?\d{2}|RX\s?[4-5]\d{2}([^\d]|$)

Reply:
i agree however that match would not necessarily be at the end due to the output of `lspci -nnk | grep -P "(VGA compatible|3D) controller" -A1`

so i would drop the `|$`

ex (although not polaris):
```
0a:00.0 VGA compatible controller [0300]: NVIDIA Corporation TU106 [GeForce RTX 2070] [10de:1f02] (rev a1)
	Subsystem: ZOTAC International (MCO) Ltd. Device [19da:2516]
--
44:00.0 VGA compatible controller [0300]: Advanced Micro Devices, Inc. [AMD/ATI] Navi 23 [Radeon RX 6600/6600 XT/6600M] [1002:73ff] (rev c1)
	Subsystem: Micro-Star International Co., Ltd. [MSI] MSI RX 6600XT MECH 2X [1462:5021]
``` 
if anything it would always be followed by a space and more parts of the name or ` [` followed by vendor:device ids

Reply:
I see many cases in the pci id list where the `|$` is needed to match. But maybe I am reading the contents wrong. Are the indented items also PCI ids or are those product names?

For example:
```
67df  Ellesmere [Radeon RX 470/480/570/570X/580/580X/590]
		1002 0b37  Radeon RX 480
		1028 1722  Radeon RX 570X
		1028 1723  Radeon RX 580X
		103c 840e  Radeon RX 580 4GB
		1043 04a8  Radeon RX 480
		1043 04b0  Radeon RX 470
		1043 04fb  Radeon RX 480
```

Reply:
the indented items are subsystem names, however i had read the regex wrong in my original message so i edited it, my apologiez.
```bash
lspci -nnk | grep -P "(VGA compatible|3D) controller" -A1
```
will yield something like, sometimes the string with the identifier is in the subsystem instead while the device name will be named something generic (although i really hope amd continues using the die name in the device name going forward like nvidia does).
```
0a:00.0 VGA compatible controller [0300]: NVIDIA Corporation TU106 [GeForce RTX 2070] [10de:1f02] (rev a1)
	Subsystem: ZOTAC International (MCO) Ltd. Device [19da:2516]
--
44:00.0 VGA compatible controller [0300]: Advanced Micro Devices, Inc. [AMD/ATI] Navi 23 [Radeon RX 6600/6600 XT/6600M] [1002:73ff] (rev c1)
	Subsystem: Micro-Star International Co., Ltd. [MSI] MSI RX 6600XT MECH 2X [1462:5021]
```

And i have done a few test cases with polaris output and the regex you made does work really well

Reply:
Well, it doesn't hurt, but in practice it may not match anything. In my previous comment I was referring to the pci id database file, not the lspci output.